protocol/client/consistencychecks.go

```diff
@@ -2,15 +2,16 @@
 // on data received from a CONIKS directory.
 // These include data binding proof verification,
 // and non-equivocation checks.
-// TODO: move all STR-verifying functionality to a separate module
 
-package protocol
+package client
```

Comment:
@masomel @Liamsi @arlolra I would like to have your inputs about this name. It is collided with our application client package (see https://github.com/coniks-sys/coniks-go/pull/183/files#diff-50a5a7c3748e6d839ee365d7f4291a1cR13). A name I think of is `verifier`. What do you think?

Reply:
Hmm. I think the auditor can also be considered a type of `verifier`, so I don't think this is the best name for this package. 

How about we change the client CLI package name? (Btw, we'll have to find a name other than `auditor` for the auditor CLI, too). Can we call it something like `client-bin` or `client-app`?

Reply:
What do you think about https://github.com/coniks-sys/coniks-go/pull/183/commits/425366425260796134bdd9baef82e6fbc6933602?